### PR TITLE
update README with implementation details

### PR DESCRIPTION
**Pull request by ksr1122**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +110,36 @@ Specific to these libraries, the simulator grader and Carla use the following:
...
+
+* /src/waypoint_updater/waypoint_updater.py
+* /src/twist_controllerd/dbw_node.py
+* /src/twist_controller/twist_controller.py
+* /src/tl_detector/tl_detector.py
+* /src/tl_detector/light_classification/tl_classifier.py
+* /src/tl_detector/sim_traffic_light_config.yaml
+* /src/tl_detector/site_traffic_light_config.yaml
+
+Additionally, `light_classifier_model` folder is added to repository which contains two sub folders i.e. *carla* and *simulation*. This folder is contains `frozen_inference_graph.pb` obtained from light classifier training model.
```
Comment (ashwinwadte) [inline README.md]: This folder `is` contains?

File: README.md [hunk 2]
```diff
@@ -107,6 +110,36 @@ Specific to these libraries, the simulator grader and Carla use the following:
...
+
+## Traffic Light Detection Node
+
+This node has the fundamental task of detecting what type of traffic signal the camera is looking at. To achieve this, it is necessary to make use of a neuronal network to classify each traffic light image into one of 4 different categories, “RED”, “GREEN”, “YELLOW” or “UNKOWN”.
```
Comment (ashwinwadte) [inline README.md]: neuronal -> neural

Comment (ashwinwadte) [reply README.md]: UNKOWN -> UNKNOWN

File: README.md [hunk 3]
```diff
@@ -107,6 +110,36 @@ Specific to these libraries, the simulator grader and Carla use the following:
...
+* /src/tl_detector/site_traffic_light_config.yaml
+
+Additionally, `light_classifier_model` folder is added to repository which contains two sub folders i.e. *carla* and *simulation*. This folder is contains `frozen_inference_graph.pb` obtained from light classifier training model.
+
+## Traffic Light Detection Node
+
+This node has the fundamental task of detecting what type of traffic signal the camera is looking at. To achieve this, it is necessary to make use of a neuronal network to classify each traffic light image into one of 4 different categories, “RED”, “GREEN”, “YELLOW” or “UNKOWN”.  
+
+In order to expedite the process and after doing some research the selection of a pre-trained model was the best for the current scenario. A pre-trained model from “Tensor flow detection model zoo” [1] library was selected. They are really useful but they still have to be adjusted in order to make it work with our current dataset of images. The selected model for our project is: ssd_mobilenet_v1_coco. The intention was to experiment with different pre-trained models, nonetheless it was found that the current model provided the performance expected for our project needs.
```
Comment (ashwinwadte) [inline README.md]: comma (,) after `some research`

Comment (ksr1122) [reply README.md]: added

File: README.md (hunk 2 above)
Comment (ksr1122) [reply README.md]: typos fixed

File: README.md (hunk 1 above)
Comment (ksr1122) [reply README.md]: removed

**Review by ashwinwadte — APPROVED** [review]

**Pull request merged**